djangocms_attributes_field/forms.py

```diff
@@ -0,0 +1,15 @@
+# -*- coding: utf-8 -*-
```

Comment:
The convention is usually forms are in a ``forms.py``, why not put this field in fields.py as well?
Has different name than the model field.